pxemgr/ipxe_handlers.go

```diff
@@ -257,6 +258,14 @@ func (mgr *pxeManagerT) imagesHandler(w http.ResponseWriter, r *http.Request) {
 		setContentLength(w, img)
 		defer img.Close()
 		io.Copy(w, img)
+	} else if strings.HasSuffix(r.URL.Path, "/qemu/coreos_production_qemu_image.img.bz2") {
```

Comment:
I actually now made it consistent with the routes and removed coreos from the url. 

this helps if you choose something else then coreos images. at least you don't have to change the part that fetches the image.